### PR TITLE
Downgrade to Ubuntu 18.04

### PR DESCRIPTION
**Pull request by twicki**
19.10 has been out of support for over 6 months, and they finally stopped hosting their package repositories.

Related: https://github.com/VulcanClimateModeling/fv3gfs-fortran/pull/105

Co-authored-by: Rhea George rheag@vulcan.com

### CODE REVIEW

**Comment by nbren12** [comment]
We already have the wrapper working with 18.04 in fv3net. You might find this dockerfile useful: https://github.com/VulcanClimateModeling/fv3net/blob/d4666f44a46df61c33cf77ba0777eefdc75ec4e0/docker/prognostic_run/Dockerfile#L55-L55.

File: Makefile [hunk 1]
```diff
@@ -31,7 +31,7 @@ export PRINT_HELP_PYSCRIPT
 
 BROWSER := python3 -c "$$BROWSER_PYSCRIPT"
 
-DOCKER_IMAGE?=us.gcr.io/vcm-ml/fv3gfs-wrapper:gnu9-mpich314-nocuda
+DOCKER_IMAGE?=us.gcr.io/vcm-ml/fv3gfs-wrapper:gnu7-mpich314-nocuda
```
Comment (mcgibbon) [inline Makefile:34]:
```suggestion
# This must be updated if the tag used in docker/Makefile is changed
DOCKER_IMAGE?=us.gcr.io/vcm-ml/fv3gfs-wrapper:gnu7-mpich314-nocuda
```

This warning comment should be enough for this PR, I'm still wondering how we can run the GPU tests with the makefile in this state.

File: tests/util.py [hunk 2]
```diff
@@ -14,7 +14,7 @@
 libc = ctypes.CDLL(None)
 c_stdout = ctypes.c_void_p.in_dll(libc, "stdout")
 
-mpi_flags = [
```
Comment (mcgibbon) [inline tests/util.py:17]: If this variable is no longer used, please remove it.

**Review by mcgibbon — APPROVED** [review]
Two small changes, after that looks good to merge!

**Pull request merged**